index.html

```diff
@@ -320,18 +320,17 @@
           view in degrees.
         </p>
         <p>
-          A <a>depth map</a> has an associated <dfn>unit</dfn> which is a
-          string. It represents the <a>active depth map unit</a>.
+          A <a>depth map</a> has an associated <dfn>precision</dfn> which is a
+          string. It represents the <a>depth precision</a> (one of the
+          <a>possible depth precisions</a>).
         </p>
         <p>
           A <a>depth map</a> has an associated <dfn>near value</dfn> which is a
-          double. It represents the minimum range in <a>active depth map
-          unit</a>s.
+          double. It represents the minimum range in millimeters.
```

Comment:
@huningxin WDYT? I'd prefer a unit for near and far values that makes the associated math simpler. The de facto unit for focal length is millimeters -- and that's what we use in this spec too.